llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -977,7 +978,7 @@ void VPWidenIntrinsicRecipe::execute(VPTransformState &State) {
 
   if (!V->getType()->isVoidTy())
     State.set(this, V);
-  State.addMetadata(V, CI);
```

Comment:
Is this change still need?

Reply:
The following patch will be used, I can remove it first. Pls give LGTM :)